projects/ngx-translate-router/src/lib/localize-router.service.ts

```diff
@@ -201,6 +201,8 @@ export class LocalizeRouterService {
    * Translate route to current language
    * If new language is explicitly provided then replace language part in url with new language
    */
+  translateRoute(path: string): string;
+  translateRoute(path: any[]): any[];
   translateRoute(path: string | any[]): string | any[] {
```

Comment:
the overloads are not needed in this TypeScript method there is a union type for the parameter and return type in the implementation signature, Why it is needed?